sass/_sass-to-js.scss

```diff
@@ -46,7 +46,7 @@
     // NUMBER
   } @else if ($valueType == 'number') {
     // '100px' has also type "number" but after multiplying to "0" gives "0px"
-    @if ('#{$value*0}' == '0') {
+    @if (unitless($value)) {
```

Comment:
Hey, @jesstelford , thanks for fixing.
It surprised me, as for Sass 3.3 and 3.4 compilers ( http://www.sassmeister.com/ )
'#{$value*0}'==0 => false
but for LibSass 3.3
'#{$value*0}'==0 => true
Good unitless works as expected